.gitignore

```diff
@@ -53,6 +53,7 @@ results-*.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+.pre-commit-config.yaml
```

Comment:
In `neptune-client` we use a `pre-commit` so go ahead and try to configure it here in a meantime :wink: ❤️